src/cmdlib.sh

```diff
@@ -206,7 +206,8 @@ EOF
         mkdir tmp/overlay-build
         (cd tmp/overlay-build && "${DIR}"/build_rpm_from_dir "${configdir}/overlay" "${name}-overlay" "${workdir}/overrides/rpm")
     fi
-    if [ -d "${overridesdir}"/rpm ]; then
+    override_rpms=$(shopt -s nullglob; echo "${overridesdir}/rpm/"*.rpm)
+    if (( ${#override_rpms} )); then
```

Comment:
Optional: huh, that's an interesting way to check it. `[ -n "$(ls ${overridesdir}/rpm/*.rpm)" ]` would also work, right? Seems less mysterious.

Reply:
Yeah, I was borrowing some code from StackOverflow, but your suggestion does look less mysterious.

Reply:
Hmm...tried multiple iterations of your suggestion to appease ShellCheck and be functional, but never found a working version.  Kept seeing variations on:

`ls: cannot access '/srv/overrides/rpm/*.rpm': No such file or directory`

I'm going to keep my original approach and add a comment with an explanation.

Reply:
Hmm, I think that's just `ls` writing to stderr in the empty case. We could silence it, though I'm good with your original patch + comment as well!

Reply:
Doh!  It looks like you are right; I mis-interpreted the `exit` (which I stuck in there to test) as a fatal error.